src/playback/qml/MuseScore/Playback/MixerPanel.qml

```diff
@@ -92,6 +103,8 @@ ColumnLayout {
                         }
 
                         prv.isPanelActivated = true
+                        let focusedIndex = item.channelItem.panel.order - 1000
+                        scrollToFocusedItem(focusedIndex)
```

Comment:
As I mentioned [here](https://github.com/musescore/MuseScore/pull/25664/files#r1876104877) there's a bug in master where `setupConnections` isn't called in time which will cause some problems for this PR. 

I can look into this separately, but in the meantime if you could make the following change that would be ideal.

``` suggestion
                        scrollToFocusedItem(i)
```

Reply:
@mathesoncalum Updated, Thanks.